src/AllMenu/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+
+class AllMenu extends Component {
+  constructor(props){
+    super(props)
+
+    this.state = {
+      dish_name:'',
+      description:''
+    }
+  }
+
+  toggleChangeMJ(){
+     this.setState(prevState => ({
+       isMJ: !prevState.isMJ,
+     }));
+   }
+
+   toggleChangeJB() {
+     this.setState(prevState => ({
+       isJB: !prevState.isJB,
+     }));
+   }
+
+   toggleChangeDrake() {
+     this.setState(prevState => ({
+       isDrake: !prevState.isDrake,
+     }));
+   }
+
+  onSubmit(e) {
+    e.preventDefault();
+   let arr = [];
+   for (var key in this.state) {
+     if(this.state[key] === true) {
+       arr.push(key);
+     }
+   }
```

Comment:
pls use filter method instead of for/in cycle

Reply:
folder was deleted